website/scripts/should-build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [[ "$VERCEL_GIT_COMMIT_REF" == "release-please--branches--main" ]] ; then
+  # Don't build
+  exit 0;
```

Comment:
This and the `exit 1` below are counterintuitive. I would've expected 0 to build and 1 to not build

Reply:
Totally fair. I think Vercel modelled this around `git diff`'s return value.

Reply:
Ah ok. That's what I was afraid of. nvm then